server/api/users.js

```diff
@@ -2,13 +2,48 @@ const router = require("express").Router();
 const { Recipe } = require("../db/models");
 const { User } = require("../db/models");
 const { UserFood } = require("../db/models");
+const { Food } = require("../db/models");
+const { ExpirationDate } = require("../db/models");
 module.exports = router;
 
+// Get all foods for a given user id
```

Comment:
The only way I can think to flatten this a bit is to associate expiration date with user_food but that's redundant. I think it's fine the way it is.

Also, is the eatBy in days?